pages/collections/explore/index.vue

```diff
@@ -0,0 +1,356 @@
+
+const { $graphql, $dataApi } = useNuxtApp()
+
+// ROUTING
+const route = useRoute()
+definePageMeta({
+  layout: 'default',
+  path: '/collections/explore',
+  alias: ['/listing-collections/explore'],
+})
+
+// ASYNC DATA
+const { data, error } = await useAsyncData('collection', async () => {
+  const data = await $graphql.default.request(COLLECTIONS_EXPLORE_LIST)
+  return data
+})
+
+if (error.value) {
+  throw createError({
+    ...error.value, statusMessage: 'Page not found.', fatal: true
+  })
+}
+
+if (!data.value.entry) {
+  throw createError({ statusCode: 404, message: 'Page not found', fatal: true })
+}
+
+// DATA
+const page = ref(_get(data.value, 'entry', {}))
+const collections = ref(_get(data.value, 'entries', []))
+const title = ref('')
+
+// PREVIEW MODE
+watch(data, (newVal, oldVal) => {
+  console.log('In watch preview enabled, newVal, oldVal', newVal, oldVal)
+  page.value = _get(newVal, 'entry', {})
+  collections.value = _get(newVal, 'entries', [])
+})
+
+// HEAD METADATA FOR THE TAB TITLES ON THE PAGE
+useHead({
+  title: page.value ? page.value.title : '... loading',
+  meta: [
+    {
+      hid: 'description',
+      name: 'description',
+      content: removeTags(page.value.summary),
+    }
+  ],
+})
+
+// COMPUTED
+const parsedCollectionList = computed(() => {
+  return collections.value.map((obj) => {
+    return {
+      ...obj,
+      to: obj.externalResourceUrl
+        ? obj.externalResourceUrl
+        : `/${obj.uri}`,
+      image: _get(obj, 'heroImage[0]image[0]', null),
+      category: obj.category.join(', '),
+      title: _get(obj, 'title', ''),
+      text: _get(obj, 'text', ''),
+    }
+  })
+})
+
+const parsedAssociatedTopics = computed(() => {
+  return page.value.associatedTopics.map((obj) => {
+    return {
+      ...obj,
+      to: obj.externalResourceUrl
+        ? obj.externalResourceUrl
+        : `/${obj.uri}`,
+    }
+  })
+})
+
+// ELASTIC SEARCH FUNCTIONALITY
+const hits = ref([])
+const noResultsFound = ref(false)
+const searchFilters = ref([])
+const searchGenericQuery = ref({
+  queryText: route.query.q || '',
+  queryFilters:
+    (route.query.filters &&
+      JSON.parse(route.query.filters)) ||
+    {},
+})
+
+// This watcher is called when router pushes updates the query params
+watch(
+  () => route.query,
+  (newVal, oldVal) => {
+    console.log('ES newVal, oldVal', newVal, oldVal)
+    searchGenericQuery.value.queryText = route.query.q || ''
+    searchGenericQuery.value.queryFilters = (route.query.filters && JSON.parse(route.query.filters)) || {}
+    searchES()
+  }, { deep: true, immediate: true }
+)
+
+// Elastic Search Function
+async function searchES() {
+  // collections.value = []
+  // hits.value = []
+  if (
+    (route.query.q && route.query.q !== '') ||
+    (route.query.filters &&
+      queryFilterHasValues(
+        route.query.filters,
+        config.exploreCollection.filters
+      ))
+  ) {
+    if (!page.title) {
```

Comment:
Follow news listing  or access searchES function. Also in the ticket the searchES does not have this logic, so remove this code 
`if (!page.title) {
      const data = await $graphql.default.request(
        COLLECTIONS_EXPLORE_LIST
      )
      page.title = _get(data, 'entry.title', '')
      page.text = _get(data, 'entry.text', '')
    }`